.github/workflows/build.yml

```
@@ -51,6 +51,50 @@ jobs:
+          # fetch-depth of 0 means to fetch the entire history not just the latest commit,
+          # and this is necessary when a workflow is triggered by something other than pushing a tag
+          fetch-depth: 0
+
+      - name: Get previous tag
+        if: steps.skip_check.outputs.should_skip != 'true'
+        id: prev_tag
+        run: |
+          echo "::set-output name=prevtag::$(git describe --tags --abbrev=0 HEAD^1)"
+
+      - name: DEBUG Set CURRENT_TAG environment variable
+        run: echo "CURRENT_TAG=foo" >> $GITHUB_ENV
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:27: Double quote to prevent globbing and word splitting

(shellcheck)